source/wp-content/themes/wporg-parent-2021/theme.json

```diff
@@ -642,24 +642,6 @@
 					"fontSize": "var(--wp--preset--font-size--normal)"
 				}
 			},
-			"core/post-content": {
```

Comment:
I was hoping `core/pattern` could have some effects on `pattern content` like `core/post-content` here does to `post content`, but it doesn't seem to work that way 🤔

Reply:
Yeah, that was my first attempt too 😞  I think because `pattern` isn't a normal block - it doesn't output any wrapper around the pattern content.

Reply:
It appears so, there's no wrapper like `wp-block-post-content` around the pattern. Not sure if GB designed it this way on purpose, or just missed it.